api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/NewCardServiceTest.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.dhsc.htbhf.claimant.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.factory.CardRequestFactory;
+import uk.gov.dhsc.htbhf.claimant.model.card.CardRequest;
+import uk.gov.dhsc.htbhf.claimant.repository.ClaimRepository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.CardRequestTestDataFactory.aValidCardRequest;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.ClaimTestDataFactory.aValidClaim;
+
+@ExtendWith(MockitoExtension.class)
+class NewCardServiceTest {
+
+    @Mock
+    private CardClient cardClient;
+
+    @Mock
+    private CardRequestFactory cardRequestFactory;
+
+    @Mock
+    private ClaimRepository claimRepository;
+
+    @InjectMocks
+    private NewCardService newCardService;
+
+    @Test
+    void shouldCallCardClientForEachNewClaim() {
+        List<UUID> claimIds = List.of(UUID.randomUUID(), UUID.randomUUID());
```

Comment:
I know this is very unlikely, but could we make these two UUIDs be constants from Strings so they will never clash and cause a build to fail.

Reply:
Seems like overkill seeing as UUID uses SecureRandom